.gitignore

```diff
@@ -7,3 +7,31 @@ coverage/*
 .frame.json
 .user.json
 .DS_Store
+
+### macOS ###
+# General
+.DS_Store
```

Comment:
Duplicate, fine to add here but remove the same above.

Reply:
Done.